bnd-maven-plugin-parent/bnd-maven-plugin-dependencies/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin-parent</artifactId>
+        <version>2.5.0-SNAPSHOT</version>
```

Comment:
Fine, I will prepare a new PR based on next. For that branch I can omit the dependency pulling stuff since it will build against Maven Central.